go/src/lib/lomcommon/helper.go

```diff
@@ -21,6 +21,8 @@ var log_level = syslog.LOG_DEBUG
 var FmtFprintf = fmt.Fprintf
 var OSExit = os.Exit
 
+var UnitTestInProgress = false
```

Comment:
Is this variable getting used in tests ? I dont see it in tx_test.go.

Reply:
Yep. I changed by overriding exit fn, which made this var unused. 
Removed